docs/guides/3_validating_public_input.md

```diff
@@ -0,0 +1,201 @@
+# Validating public input
+
+You can validate that the public input of the proof sent to Aligned for verification is correct in a few simple steps.
+
```

Comment:
I would add a little more context on why we would like to do this. For example:

> In some applications, you would like to attest that other party has done some computation correctly and actually do something with the result of that computation. For this, the other party should have interacted with Aligned first and obtained the `AlignedVerificationData`, a receipt indicating that the proof of the computation was verified correctly.
The application should receive the Aligned verification data as well as the result of the computation. After validating that the proof was verified in Aligned, it should check that the posted result is indeed the same as the one commited in the Aligned verification data.

Something similar to that should give a bit more context to why is this important

Reply:
Solved in [098bd93](https://github.com/yetanotherco/aligned_layer/pull/648/commits/098bd93274cdae2f9d06a90212af20c96b65ceea)